addons/website/static/src/js/editor/widget_link.js

```diff
@@ -33,7 +39,11 @@ weWidgets.LinkTools.include({
             },
             body: this.$editable[0].ownerDocument.body,
         };
-        wUtils.autocompleteWithPages(this, this.$('input[name="url"]'), options);
+        const fakeWidget = {
+            _rpc: ({ route, params }) => this.rpc(route, params),
+            trigger_up: this._onAutocompleteClose.bind(this),
+        };
```

Comment:
`fakeWidget`, what ? Oh, like a very limited ultra ad-hoc compatibility layer thing ? Maybe add a comment for that.